docs/types/value-types.rst

```diff
@@ -656,13 +656,8 @@ External (or public) functions have the following members:
 
 Example that shows how to use the members::
 
-<<<<<<< HEAD
-    pragma solidity >=0.6.4 <0.8.0;
-=======
     // SPDX-License-Identifier: GPL-3.0
-    pragma solidity >=0.6.0 <0.7.0;
-    // This will report a warning
```

Comment:
Don't we need that comment?

Reply:
The comment was about the old `.gas()` and `.value()` syntax.

This was the old code: 
```solidity
contract Example {
    function f() public payable returns (bytes4) {
        assert(this.f.address == address(this));
        return this.f.selector;
    }

    function g() public {
        this.f.gas(10).value(800)();
        // New syntax:
        // this.f{gas: 10, value: 800}()
    }
}
```

Reply:
Somehow it passes the tests without comment. With the comment it was failing tests. We should check why it happened anyway

Reply:
@mijovic  It doesn't pass without that comment because there is no warning, and the CI expects a warning in that snippet.

Reply:
For a comparison see: https://github.com/ethereum/solidity/pull/8589/commits/bd0b06e8dbb86252357a0771be25b9879731caba#diff-4d08fe4d02689940a95bc8204f9cf476